pmd-tsql/src/main/antlr4/net/sourceforge/pmd/lang/tsql/antlr4/TSqlLexer.g4

```diff
@@ -0,0 +1,1283 @@
+/*
```

Comment:
Please move this into the package `ast` instead of `antlr4`: `net/sourceforge/pmd/lang/tsql/ast`